khal/settings/khal.spec

```diff
@@ -76,6 +76,14 @@ type = option('calendar', 'birthdays', 'discover', default='calendar')
 # belongs to the user.
 addresses = force_list(default='')
 
+# The address adapter is a command that will be run using "bash -c" to get
+# a list of addresses for autocompletion in the attendee field of a new event
+# in interactive mode. This expects the same output as "khard email | tail -n +2"
+# since it has only been developed with khard in mind - but other providers
+# should be configurable to create the same output, or the parser could
+# be extended (see _contacts_update in CalenderCollection in khalendar.py)
+address_adapter = string(default=None)
```

Comment:
What behaviour does `None` cause and why not default to `khard email | tail -n +2`?

Reply:
`None` causes no autocompletion to be available (no address book). The idea behind not setting the default to `khard email` was that not everyone might have khard installed.

However, I agree that this is counterintuitive. I have committed a change that:
* Adds an address_adapter field to the [default] section that
  * Is, by default, `khard email | tail -n +2`
  * Can be `None`, meaning "empty address book"
* Adds the possible value "default" (which is also the default) to the address_adapter in the account section. That value causes the address_adapter from the [default] section to be used for that calendar
* Tests whether the command from address_adapter was run successfully. If that is not the case, the effect is the same as setting address_adapter to `None`, i.e. the address book is assumed to be empty.

Does that make more sense?